skunkworks_crow/src/main/java/org/odk/share/views/ui/main/MainActivity.java

```diff
@@ -107,6 +107,15 @@ protected void onCreate(Bundle savedInstanceState) {
         addListItemDivider();
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        //update the list
```

Comment:
The code is very clear and we don't need this comment.

Reply:
Yeah, I updated.
can you please have a look at my previous [PR's](https://github.com/opendatakit/skunkworks-crow/pulls/Chromicle)